docs/source/additional/faq.rst

```diff
@@ -5,27 +5,26 @@ FAQ
-
-Substra is the most proven software for federated learning on healthcare data in real production environments. It has already been deployed and used by hospitals and biotech companies (see the `MELLODDY <https://www.melloddy.eu/>`_ project for instance). Substra can also be used on a single machine on a virtually splitted dataset to perform FL simulations and debug code before launching experiments on a real network.
+Substra is an open source federated learning (FL) software that enables machine learning on distributed datasets. It provides a flexible Python interface and a web app to perform federated machine learning at scale.
 
+Substra is the most proven software for federated learning in healthcare and has already been deployed in real production environments by hospitals and biotech companies (see the `MELLODDY <https://www.melloddy.eu/>`_ project for instance). Substra can also be used on a single machine on a virtually splitted dataset to perform FL simulations and debug code before launching experiments on a real network.
 
 Who owns Substra?
 ^^^^^^^^^^^^^^^^^
-Substra was originally developed by `Owkin <https://owkin.com/>`_ and is now hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Today Owkin is the main contributor to Substra.
+Substra is open source tool operated under an Apache 2.0 License. Substra is hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_. Substra was initially developed by engineers at `Owkin <https://owkin.com/>`_, a BioTech company that continues to play a big role in its development.
 
 What kinds of data does Substra support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Substra can run tasks on any type of data: tabular data, images, videos, audio, time series, etc.
 
 What kind of machine learning model can I use with Substra?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Substra is fully compatible with machine learning models written in Python from any library (PyTorch, Tensorflow, Sklearn, etc). However, a specific interface has been developed to use PyTorch in Substra, which makes writing PyTorch code simpler than using other frameworks.
+Substra is fully compatible with machine learning models written in any Python library (PyTorch, Tensorflow, Sklearn, etc). However, a specific interface has been developed to use PyTorch in Substra, which makes writing PyTorch code simpler than using other frameworks.
 
 Is Substra limited to medical and biotech applications?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Substra has been designed to work especially well on healthcare use cases; however, as Substra can work on any kind of data with any Python libraries, Substra can be used for any computation on distributed data.
+Although substra has been designed to work especially well in healthcare settings, it can work on any kind of data with any Python library to perform computation or analysis using distributed data. 
```

Comment:
```suggestion
Although Substra has been designed to work especially well in healthcare settings, it can work on any kind of data with any Python library to perform computation or analysis using distributed data. 
```